articles/app-service-mobile/app-service-mobile-xamarin-forms-get-started-push.md

```diff
@@ -196,7 +196,8 @@ With the back end configured with FCM, you can add components and codes to the c
         {
             var intent = new Intent(this, typeof(MainActivity));
             intent.AddFlags(ActivityFlags.ClearTop);
-            var pendingIntent = PendingIntent.GetActivity(this, 0, intent, PendingIntentFlags.OneShot);
+            //Random number the second Parameter
+            var pendingIntent = PendingIntent.GetActivity(this, UNIQUE_INT_PER_CALL, intent, PendingIntentFlags.OneShot);
```

Comment:
To make the code snippet compilable, we can use a random number generator like this:

```suggestion
            var requestCode = new Random().Next();
            var pendingIntent = PendingIntent.GetActivity(this, requestCode, intent, PendingIntentFlags.OneShot);
```

Reply:
I made this update.